bin/gen_csr_access_test.py

```diff
@@ -148,6 +220,11 @@ def preprocess_yaml():
     if not args.dry_run:
         output_script_handle = open(output_script_path, "w")
 
+    # "m4" - Optionally used for all preprocessing
+    if (args.m4):
+        preprocess_yaml_m4(enabled_features, input_script_path, output_script_handle)
```

Comment:
Invocation of `m4`.

Reply:
If we are going to use `m4` then we need to update [requirements.txt](https://github.com/openhwgroup/core-v-verif/blob/6113f4f179645eb05c4178b6ec25be316a9a1aa8/bin/requirements.txt#L1), and probably also the top-level README.

Reply:
All references I can find to `requirements.txt` indicate that this is used for Python packages: e.g. `pip install -r requirements.txt`. Is that correct?

But `m4` is not a Python package, it is a standard posix command line program along with `grep`, `ls`, `cd`, `chmod`, `cp`, `echo`, `make`, etc.
https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/idx/utilities.html
_([main link](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/))_